peps/pep-0771.rst

```diff
@@ -79,9 +124,9 @@ the community for several years, including in `this DPO thread
 as well as in numerous issues and pull requests. The solution that will be
 presented below:
 
-* does not break backward-compatibility of existing packaging infrastructure
 * is an opt-in solution which means that package maintainers can choose whether or not to use it
-* is flexible enough to accommodate both of the major use cases described in `Motivation`_.
+* is flexible enough to accommodate both of the major use cases described in `Motivation`_
+* does not introduce new syntax compared to :pep:`508` (which would have serious backward-compatibility implications)
```

Comment:
"serious backward-compatibility implications" here is unqualified, I would expect an explanation or a link to a section where it is justified.
```suggestion
* re-uses the syntax from :pep:`508`
```